include/aspect/parameters.h

```diff
@@ -319,6 +319,7 @@ namespace aspect
     bool                           use_conduction_timestep;
     bool                           convert_to_years;
     std::string                    output_directory;
+    std::string                    world_builder_directory;
```

Comment:
Hmm, why do you need this variable? Could it not be a member of the world builder?